### ISSUE TITLE
Teams installs not working

### ISSUE THREAD

**Issue by dannyboyayyo** [Deyda/NeverRed#78]
Teams classic is not being installed, missing from add/remove programs and New Teams does not show up either (Windows 11)

Using FSLogix 2210 hotfix 3 (2.9.8784.63912) and windows 11 also server 2019. I know there are still issues with New Teams and server 2019 but the classic is also not installing suddenly

**Comment by chezzer64** [comment]
Line 20873 needs updating:
```
        $TeamsNewD = Get-EvergreenApp -Name MicrosoftTeamsPreview -WarningAction silentlyContinue | Where-Object { $_.Architecture -eq "$MSTeamsNewArchitectureClear" -and $_.Release -eq "Enterprise" -and $_.Type -eq "msix"}
```
MicrosoftTeamsPreview should be changed to MicrosoftTeams - following the Teams Application renames in Evergreen.

**Comment by chezzer64** [comment]
Also, I think there may be an issue with Teams Meeting Addin logic for New Teams:
```
PS > $appDLLs = (Get-ChildItem -Path "C:\Windows\Microsoft\TeamsMeetingAddin\x64" -Include "Microsoft.Teams.AddinLoader.dll" -Recurse).FullName
PS > $appX64DLL = $appDLLs[0]
PS > $appDLLs
C:\Windows\Microsoft\TeamsMeetingAddin\x64\Microsoft.Teams.AddinLoader.dll
PS > $appX64DLL
C
PS >
```
In my environment ```$appDLLs``` is returned as a string (as only 1 DLL is found) and not an array, therefore ```$appX64DLL``` is set to just the first character of the filename ('C'). Likewise for ```$appX86DLL```.

**Comment by Deyda** [comment]
Solved in 2.10.28
